common/service/sensor/sensor.c

```diff
@@ -806,6 +810,16 @@ bool sensor_init(void)
 	init_sensor_num();
 	// Check init SDR size is equal to sensor config size
 	check_init_sensor_size();
+	if (pdr_config_size != 0) {
+		numeric_sensor_table =
+			(PDR_numeric_sensor *)malloc(pdr_config_size * sizeof(PDR_numeric_sensor));
+		if (numeric_sensor_table != NULL) {
+			pdr_init();
+		} else {
+			LOG_ERR("numeric_sensor_table == NULL");
```

Comment:
Do you want to continue on failure?

Reply:
Most platforms don't have PDR table now, so we make this continue on failure. We plan to implement another sensor service for PLDM PDR sensors. I leaved a TODO for indication.